snippets/loader-component.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Loader Component
+level: beginner
+author: thekiba
+tags:
+  - tips
+  - good-to-know
+  - components
+  - templates
+---
+
+# Content
+You can create own helper component and use it instead of `*ngIf`.
+
+```typescript
+@Component({
+  selector: 'loader',
+  template: `
+    <ng-content *ngIf="!loading else showLoader"></ng-content>
+    <ng-template #showLoader>🕚 Wait 10 seconds!</ng-template>
+  `
+})
+class LoaderComponent {
+  @Input() loading: boolean;
+}
+```
+
+For usage example:
+```html
+<loader [loading]="isLoading">🦊 🦄 🐉</loader>
```

Comment:
Maybe use structural directive?
<loader [loading]="isLoading">
   <destroy-the-world></destroy-the-world>
</loader>

Reply:
Or add a note